### PR TITLE
Unit test closed sink

### LINKED ISSUE #31: Unit test for the "Closed" Sink
Create a unit test for the [Closed](https://github.com/ReactiveHTML/rimmel/blob/master/src/sinks/closed-sink.ts) sink.

Please make sure test cases are in BDD styles matching the other sinks.

### PR DESCRIPTION
**Pull request by TanmayRanaware**
fixes #31 
Test coverage:
ClosedSink function and Closed explicit sink
Dialog element closing functionality
Boolean attribute values (true/false)
Promise and observable sources
Edge cases and integration scenarios

### CODE REVIEW

File: src/sinks/closed-sink.test.ts [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+            sink([]);
+
+            // Test with various falsy values
+            sink(false);
+            sink(0);
+            sink('');
+            sink(null);
+            sink(undefined);
+
+            expect(dialog.close).toHaveBeenCalledTimes(10);
```
[the remark this answers is not in the document]
Comment (TanmayRanaware) [reply src/sinks/closed-sink.test.ts]: got it . The tests now follow the one action => one reaction principle.

File: src/sinks/closed-sink.test.ts [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+            expect(config.t).toBe(CLOSED_SINK_TAG);
+            expect(config.source).toBe(source);
+            expect(config.sink).toBe(ClosedSink);
+        });
+
+        it('preserves source reference in configuration', () => {
+            const source = false;
+            const config = Closed(source);
+
+            expect(config.source).toBe(source);
```
[the remark this answers is not in the document]
Comment (TanmayRanaware) [reply src/sinks/closed-sink.test.ts]: yes

File: src/sinks/closed-sink.test.ts [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+            const sink = ClosedSink(dialog);
+
+            expect(typeof sink).toBe('function');
+            expect(dialog.close).not.toHaveBeenCalled();
+
+            sink(true);
+            expect(dialog.close).toHaveBeenCalledTimes(1);
+
+            sink(false);
+            expect(dialog.close).toHaveBeenCalledTimes(2);
```
[the remark this answers is not in the document]
Comment (TanmayRanaware) [reply src/sinks/closed-sink.test.ts:45]: done

**Comment by dariomannu** [comment]
Amazing, that's merged now!

File: src/sinks/closed-sink.test.ts [hunk 4]
```diff
@@ -54,7 +55,7 @@ describe('Closed Sink', () => {
                 expect(dialog.close).toHaveBeenCalledTimes(1);
             });
 
-            it('closes dialog when given number 1', () => {
+            it('calls dialog.close() when given number 1', () => {
```
[the remark this answers is not in the document]
Comment (TanmayRanaware) [reply src/sinks/closed-sink.test.ts]: done

File: src/sinks/closed-sink.test.ts [hunk 5]
```diff
@@ -217,10 +218,11 @@ describe('Closed Sink', () => {
         });
 
         describe('when called multiple times', () => {
```
[the remark this answers is not in the document]
Comment (TanmayRanaware) [reply src/sinks/closed-sink.test.ts]: done
